packages/compass-components/src/hooks/use-theme.tsx

```diff
@@ -50,8 +51,7 @@ const withTheme = function <
     const theme = useTheme();
 
     const applyTheme =
-      theme.enabled === true ||
-      global?.process?.env?.COMPASS_LG_DARKMODE === 'true';
+      theme.enabled === true || preferences.getPreferences().lgDarkmode;
```

Comment:
@gribnoysup Yes, all of this absolutely makes sense! Can you take a look at 4cd9bf2bf0b9ed6e62c4ba242ee4f6e4e748d6b8 and see if that matches what you were expecting? I’m just feeling a little confused about it because I couldn’t find anything setting the `enabled` flag before, but maybe there really wasn’t anything.

Reply:
I am in the same boat and it also took me a moment to grasp how these flags should interact 😄 But yeah, at least I can definitely say that our understanding is the same here, this commit looks good to me and what I would expect!

I think `enabled` is currently exclusively used in the Tooltip component to invert the theme for anything that is rendered inside the tooltip, because they are dark in the light mode and vice verca:

https://github.com/mongodb-js/compass/blob/1326e454c989aabae8f884b9e3a81fb2fc836f5c/packages/compass-components/src/components/tooltip.tsx#L68-L76